docker/nginx.conf

```diff
@@ -230,7 +230,7 @@ http {
             proxy_set_header X-Real-IP $remote_addr;
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded-Proto $scheme;
-            proxy_pass http://dashflux:3000/;
+            proxy_pass http://dashflux.default:3000/;
```

Comment:
In the last k8s pull request (previous to my pull request) it was decided to have three different namespaces: grpc-services, ingress-nginx and default.
Things, Users and the different adapters are declared under grpc-services and the rest of services (like dashflux) are created under the default namespace. Ingress-nginx was created, but it was not finally used.
So just in case someone decides to use nginx and associate it to ingress-nginx namespace, I included the default namespace for dashflux. That way, if someone creates nginx under ingress-nginx namespace, it will work without having to change nginx.conf; and if he/she decides to create it under default namespace (as it is today), it will also work (even the information is redundant)

Reply:
Note now that I have created a new nginx.conf file for k8s deployment, "http://dashflux.default:3000/" will be only applicable to k8s and not to docker-compose.
The explanation of why "default" is included is shown in my comment above and it is related to a previous pull request.